src/main/java/net/sf/jsqlparser/schema/Sequence.java

```diff
@@ -1,180 +1,223 @@
-        setIndex(SCHEMA_IDX, string);
-    }
-
-    public String getName() {
-        return getIndex(NAME_IDX);
-    }
-
-    public void setName(String string) {
-        setIndex(NAME_IDX, string);
-    }
-
-    private void setIndex(int idx, String value) {
-        int size = partItems.size();
-        for (int i = 0; i < idx - size + 1; i++) {
-            partItems.add(null);
-        }
-        partItems.set(idx, value);
-    }
-
-    private String getIndex(int idx) {
-        if (idx < partItems.size()) {
-            return partItems.get(idx);
-        } else {
-            return null;
-        }
-    }
-
-    @Override
-    public String getFullyQualifiedName() {
-        StringBuilder fqn = new StringBuilder();
-
-        for (int i = partItems.size() - 1; i >= 0; i--) {
-            String part = partItems.get(i);
-            if (part == null) {
-                part = "";
-            }
-            fqn.append(part);
-            if (i != 0) {
-                fqn.append(".");
-            }
-        }
-
-        return fqn.toString();
-    }
-
-    @Override
-    public String toString() {
-        StringBuilder sql = new StringBuilder(getFullyQualifiedName());
-        if (parameters != null) {
-            for (Sequence.Parameter parameter : parameters) {
-                sql.append(" ").append(parameter.formatParameter());
-            }
-        }
-        return sql.toString();
-    }
-
-    /**
-     * The available parameters to a sequence
-     */
-    public enum ParameterType {
-        INCREMENT_BY,
-        START_WITH,
-        MAXVALUE,
-        NOMAXVALUE,
-        MINVALUE,
-        NOMINVALUE,
-        CYCLE,
-        NOCYCLE,
-        CACHE,
-        NOCACHE,
-        ORDER,
-        NOORDER,
-        KEEP,
-        NOKEEP,
-        SESSION,
-        GLOBAL
-    }
-
-    /**
-     * Represents a parameter when declaring a sequence
-     */
-    public static class Parameter {
-        private final ParameterType option;
-        private Long value;
-
-        public Parameter(ParameterType option) {
-            this.option = option;
-        }
-
-        public Long getValue() {
-            return value;
-        }
-
-        public void setValue(Long value) {
-            this.value = value;
-        }
-
-        public String formatParameter() {
-            switch (option) {
-                case INCREMENT_BY:
-                    return withValue("INCREMENT BY");
-                case START_WITH:
-                    return withValue("START WITH");
-                case MAXVALUE:
-                case MINVALUE:
-                case CACHE:
-                    return withValue(option.name());
-                default:
-                    // fallthrough just return option name
-                    return option.name();
-            }
-        }
-
-        private String withValue(String prefix) {
-            return prefix + " " + value;
-        }
-    }
-}
+/*
```

Comment:
Why is this file completely replaced? Are there some kind of linefeed differences?

Reply:
https://github.com/JSQLParser/JSqlParser/pull/1014/commits/324ccf112f199eadc5a936f285130d9c929eeb0b
tried to replace formatchanges, that compare finds matching lines

Reply:
https://github.com/JSQLParser/JSqlParser/pull/1014/commits/0d434b81b47a3cb2b24eb7487ae8d6fecbf0d1cd
seems that the original commit had different linefeeds than the other files in this project

Reply:
I',m sorry, did't work for this diff-view

Reply:
I think that was me. I work on a Mac and on a windows device from time to time. I’ll double check that we have a gitatttrbutes setup correctly to avoid these issues!

Reply:
Unfortunately only the latest versions of jgit, if you are using this, are taking gitattributes into account. But that explains this. If the file was wrong, now it would be corrected. So all fine.

Reply:
AH, I use intellij, so i bet my Windows gitbash auto-core config crap wasn't setup right.